addon/display/panel.js

```diff
@@ -54,6 +54,7 @@
     this.options = options;
     this.height = height;
     this.cleared = false;
+    window.addEventListener("resize", () => this.changed());
```

Comment:
This leaks an event listener. You could make the handler self-unregister when the editor is no longer in the DOM, but it might be cleaner to register a [`"refresh"` event](https://codemirror.net/doc/manual.html#event_refresh) handler, which the editor will call when it is resized or refreshed.

Reply:
thanks for the hint, I asked myself how to clean this up already, but didn't think of CodeMirrors event system...

Reply:
Plugins are not getting compiled, are they?
Using an arrow function is probably a bad idea then, right?